@here/harp-mapview/lib/ConcurrentWorkerSet.ts

```diff
@@ -157,7 +158,8 @@ export class ConcurrentWorkerSet {
         const workerCount = getOptionValue(
             this.m_options.workerCount,
             typeof navigator !== "undefined" && navigator.hardwareConcurrency !== undefined
-                ? Math.max(1, navigator.hardwareConcurrency) // we need to have at least one worker
+                ? // We need to have at least one worker
+                  MathUtils.clamp(navigator.hardwareConcurrency - 1, 1, 4)
```

Comment:
Why do you subtract one from the concurrency?

Reply:
So one thread is left for rendering.